apps/desktop/src/components/editor-area/annotation-box.tsx

```
@@ -0,0 +1,329 @@
+              <div className="flex items-center gap-2 text-neutral-500">
+                <Loader2 className="h-3 w-3 animate-spin" />
+                <span className="text-xs">Analyzing transcript...</span>
+              </div>
+            )
+            : (
+              <div className="text-xs text-neutral-700 leading-relaxed h-full overflow-y-auto">
+                {streamedContent
+                  ? (
+                    <div
+                      className="prose prose-xs max-w-none"
+                      dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)